docs/standard/exceptions/handling-com-interop-exceptions.md

```diff
@@ -22,6 +22,13 @@ Managed and unmanaged code can work together to handle exceptions. If a method t
   
  If a method fails in unmanaged code, an exception can be passed to a managed code segment. The topic [HRESULTS and Exceptions](../../../docs/framework/interop/how-to-map-hresults-and-exceptions.md) contains a table showing how HRESULTS map to runtime exception objects.  
```

Comment:
This topic is about exception handling and COM. It is all Windows specific. This should be in the part of documentation that talks about platform-neutral behaviors. Maybe there should be a new top level file that talks about exception handling interop?

Reply:
I think we should create a new file in this case. I glanced over the doc files under _native-interop_ and all of them refer to COM interop and other things. On the other hand, the _exceptions_ directory has several how to's, but the only interop file is the one I modified here. Maybe we could name it something like `interop-exceptions-limitations` or `interop-exceptions-handling`?

Reply:
I like `interop-exceptions-handling`. Or make it just `interop-exceptions`

Reply:
I just realized the current file is called `handling-com-interop-exceptions.md`. I think it might be a bit confusing to name the new one just `interop-exceptions`. Maybe something like `platform-neutral-interop-exceptions.md` would work?

Reply:
None of the platform-neutral docs under https://docs.microsoft.com/en-us/dotnet/standard/native-interop/ has platform-neutral prefix, so I do not think this one should  have it either.

Reply:
Fair point. Will change it.